### PR TITLE
Add Japanese trasnlation of chapter 1 / 2 to 3

### PR DESCRIPTION
**Pull request by blackdoor571**

### CODE REVIEW

**Comment by ArthurZucker** [comment]
Awesome work! 🤗 
In order to have a preview, you need to add the chapters to the `chapters/ja/_toctree.yml` as in #363 👍🏻 
Then we'll be able to make sure it renders well

File: chapters/ja/chapter1/3.mdx [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+
+このpipelineは、提供されたコンテキストから情報を抽出することで動作し、答えを生成するわけではないことに注意してください。
+
+## 要約
+
+要約とは、文章中の重要な部分をすべて（あるいはほとんど）維持したまま、より短い文章にするタスクです。以下はその例です。
+
+```python
+from transformers import pipeline
+summarizer = pipeline("summarization")
```
Comment (fxmarty) [inline chapters/ja/chapter1/3.mdx:264]: maybe there is a model on the Hub for summarization in Japanese? maybe https://huggingface.co/csebuetnlp/mT5_multilingual_XLSum ? Tried on the first paragraphs from https://www3.nhk.or.jp/news/html/20221104/k10013881401000.html , it works not too bad :D

File: chapters/ja/chapter1/3.mdx [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+
+
+## 翻訳
+
+翻訳の場合、タスク名に言語ペアを指定すれば（`"translation_en_to_fr"`など）デフォルトのモデルを使うこともできますが、一番簡単なのは [Model Hub](https://huggingface.co/models) で使いたいモデルを選ぶことです。ここでは、フランス語から英語への翻訳を試してみます。
+
+```python
+from transformers import pipeline
+translator = pipeline("translation", model="Helsinki-NLP/opus-mt-fr-en")
+translator("Ce cours est produit par Hugging Face.")
```
Comment (fxmarty) [inline chapters/ja/chapter1/3.mdx:308]: maybe japanese to english model here?

File: chapters/ja/chapter1/3.mdx [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+
+</Tip>
+
+## 固有表現認識
+
+固有表現認識（NER）は、入力されたテキストのどの部分が人物、場所、組織などの固有表現に対応するかをモデルが見つけ出すタスクです。例を見てみましょう。
+
+```python
+from transformers import pipeline
+ner = pipeline("ner", grouped_entities=True)
```
Comment (fxmarty) [inline chapters/ja/chapter1/3.mdx:218]: maybe https://huggingface.co/ken11/bert-japanese-ner could be nice here?

File: chapters/ja/chapter1/3.mdx [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+
+## pipelineでHubから任意のモデルを使用する
+
+これまでの例では、タスクに応じたデフォルトのモデルを使用しましたが、特定のタスク（例えばテキスト生成）のpipelineで使用するモデルをHubから選択することも可能です。[Model Hub](https://huggingface.co/models)にアクセスし、左側の対応するタグをクリックすると、そのタスクでサポートされているモデルのみが表示されます。[このようなページ](https://huggingface.co/models?pipeline_tag=text-generation)が表示されるはずです。
+
+それでは、[`distilgpt2`](https://huggingface.co/distilgpt2)モデルを試してみましょう! 先ほどと同じpipelineでロードする方法を説明します。
+
+```python
+from transformers import pipeline
+generator = pipeline("text-generation", model="distilgpt2")
```
Comment (fxmarty) [inline chapters/ja/chapter1/3.mdx:150]: maybe we could use one model from https://huggingface.co/models?language=ja&pipeline_tag=text-generation&sort=downloads ?

**Comment by fxmarty** [comment]
This is super cool ️‍🔥 I was thinking it could be nice to have examples in Japanese throughout the course, what do you think? Just a suggestion!

File: chapters/ja/chapter1/3.mdx (hunk 4 above)
Comment (lewtun) [reply chapters/ja/chapter1/3.mdx:150]: I think it's best to leave the source files with English models / outputs and translate the notebooks themselves with Japanese models (if desired).

This has the advantage of ensuring parity between the English <--> Japanese translations and means you don't have to reword a lot of text to explain e.g. the inputs / outputs.

See [here](https://github.com/huggingface/notebooks/pull/247) for an example of how this was done for the French translation (but it's certainly not a requirement for this great translation effort!)

**Review by lewtun — APPROVED** [review]
Thanks a lot for this great translation @blackdoor571 and welcome to the 🤗 community!

I think @fxmarty's idea to use Japanese models is nice, but probably best done directly in the notebooks to ensure the course content is 1-1 across languages. No need to do that yourself, but just making a note here for other translators :)

There's a small code styling issue with the PR - could you please install the requirements and run `make style`?

File: chapters/ja/chapter1/3.mdx (hunk 4 above)
Comment (fxmarty) [reply chapters/ja/chapter1/3.mdx:150]: Yes it makes sense!

**Comment by fxmarty** [comment]
@lewtun Makes sense. Here's an example of how it was done in the French translation, with comments: https://huggingface.co/course/fr/chapter1/3?fw=pt . Honestly I'm not fan of putting comments everywhere (everything feels like less readable), but I see the point where there may be readers not able to grasp examples in English.

**Comment by lewtun** [comment]
Hey @blackdoor571 thanks for this great translation! We just merged #359 which created some merge conflicts on the `_toctree.yml` file. Would you mind rebasing / merging with `main` and pushing again? Thanks!

**Pull request merged**